examples/sites/demos/pc/webdoc/changelog.md

```
@@ -4,6 +4,173 @@ Tiny Vue 团队在正常情况下使用 每月 发布策略。
 
 在此页面上，您只能看到我们的 更新日志 最新三个迭代的日志记录，如您要查看完整记录可以查看：[Release](https://github.com/opentiny/tiny-vue/releases)
 
+## v3.26.0/v2.26.0
+
+`2025/09/15`
+
+## What's Changed
+
+### Exciting New Features 🎉
+
+- feat(steps): Add wizard style step bar itemStyle differentiated configuration by @chenxi-20 in https://github.com/opentiny/tiny-vue/pull/3594
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)